editor.planx.uk/src/@planx/components/Checklist/Public/tests/mockOptions.ts

```diff
@@ -168,3 +169,585 @@ export const groupedOptionsWithExclusiveOption: Array<Group<Option>> = [
     ],
   },
 ];
+
+export const mockWithRepeatedOptions: PublicChecklistProps = {
```

Comment:
I'm also keen to have this mock within Storybook as it's one of the larger checklists - should be useful for testing and refactoring improvements to the Editor modal (e.g. possibly swapping out Formik, or otherwise optimising performance)/